addons/event/models/event_mail.py

```diff
@@ -96,6 +96,7 @@ def _compute_scheduled_date(self):
 
     @api.one
     def execute(self):
+        now = datetime.strftime(fields.datetime.now(), tools.DEFAULT_SERVER_DATETIME_FORMAT)
```

Comment:
we can even use `fields.Datetime.now()` instead of the whole `strftime`. This is a bit misleading by the way, as `fields.datetime == datetime.datetime` and `fields.Datetime` is the Odoo wrapper.